src/Sql/Sql/Database Backup/Model/AzureSqlDatabaseLongTermRetentionPolicyModel.cs

```diff
@@ -36,6 +36,6 @@ public class AzureSqlDatabaseLongTermRetentionPolicyModel
         /// <summary>
         /// Gets or sets the long term retention policy of the database
         /// </summary>
-        public BackupLongTermRetentionPolicy Policy { get; set; }
+        public LongTermRetentionPolicy Policy { get; set; }
```

Comment:
why do you rename this?

Reply:
I think I just changed it to match the swagger/SDK name

Reply:
But this is a good point there is no need for this to be renamed

Reply:
Actually, this file is never used.  AzureSqlDatabaseBackupLongTermRetentionPolicy.cs is used instead.  I'll delete this